docs/source/federation-versions.mdx

```diff
@@ -25,6 +25,90 @@ For a comprehensive changelog for Apollo Federation and its associated libraries
 
 - If you maintain a [subgraph-compatible library](./building-supergraphs/supported-subgraphs/), consult this article to stay current with recently added directives. All of these directive definitions are also listed in the [subgraph specification](./subgraph-spec/#subgraph-schema-additions).
 
+## v2.8
+
+<hr/>
+
+<CodeColumns cols="3">
+
+<div>
+
+First release
+
+**May 2024**
+
+</div>
+
+<div>
+
+Available in GraphOS?
+
+**No**
+
+</div>
+
+<div>
+
+Minimum router version
+
+**1.48.0**
+
+</div>
+
+</CodeColumns>
+
+<hr/>
+
+#### Directive changes
+
+<table>
+  <thead>
+    <tr>
+      <th>Topic</th>
```

Comment:
I [previously introduced](https://github.com/apollographql/federation/pull/3013) this so directive names weren't scrunched. Annoying to have to repeat--might just make a class for all the tables on this page.

```suggestion
      <th style={{ minWidth: 200 }}>Topic</th>
```